lua/neotest-jest/init.lua

```diff
@@ -47,19 +47,31 @@ end
 function adapter.discover_positions(path)
   local query = [[
     ; -- Namespaces --
-    ; Matches: `describe('context')`
+    ; Matches: `describe('context', () => {})`
     ((call_expression
       function: (identifier) @func_name (#eq? @func_name "describe")
       arguments: (arguments (string (string_fragment) @namespace.name) (arrow_function))
     )) @namespace.definition
-    ; Matches: `describe.only('context')`
+    ; Matches: `describe('context', function() {})`
```

Comment:
Originally I tried changing the previous matcher to use `[(arrow_function) (function)]` but this didn't seem to work correctly. I'm not very familiar with treesitter but pulling this out separately seemed to work

Reply:
You can write something like this 
```
arguments: (arguments (string (string_fragment) @test.name) (_) @function_type (#has-type? @function_type "function" "arrow_function"))
```
and it will match both `function` and `arrow_function` without duplicating queries

Reply:
Hey @KostkaBrukowa, thanks for looking at this! I tried this out locally but am seeing this error:

```
Fail	||	build_spec with override builds command
            ../plenary.nvim/lua/plenary/async/util.lua:37: Blocking on future timed out or was interrupted.
            ...im/0.8.2/share/nvim/runtime/lua/vim/treesitter/query.lua:495: No handler for has-type?

            stack traceback:
            	../plenary.nvim/lua/plenary/async/util.lua:37: in function 'block_on'
            	../plenary.nvim/lua/plenary/async/util.lua:48: in function <../plenary.nvim/lua/plenary/async/util.lua:47>
```

Any ideas what would cause this?

Reply:
hmmm... Do you have `nvim-treesitter` installed? - this plugin adds this query here https://github.com/nvim-treesitter/nvim-treesitter/blob/master/lua/nvim-treesitter/query_predicates.lua#L87. I can see that `nvim-treesitter` is  a requirement for neotest.

Reply:
I'm using `script/test` which looks like it might be missing some setup steps. If I run the tests using my normal instance of nvim I get this error:

```
Fail    ||      build_spec with override builds command
            .../share/nvim/lazy/plenary.nvim/lua/plenary/async/util.lua:37: Blocking on future timed out or was interrupted.
            ...re/nvim/lazy/neotest/lua/neotest/lib/treesitter/init.lua:36: attempt to index local 'definition' (a nil value)
            
            stack traceback:
                .../share/nvim/lazy/plenary.nvim/lua/plenary/async/util.lua:37: in function 'block_on'
                .../share/nvim/lazy/plenary.nvim/lua/plenary/async/util.lua:48: in function <.../share/nvim/lazy/plenary.nvim/lua/plenary/async/util.lua:47>
```

This is the same error I was getting when I used `[(arrow_function) (function)]`

Reply:
So if it doesn't work for you then it wouldn't work for someone else probably. personally I wouldn't bother with it and stay with current approach